sql/item_mods.sql

```diff
@@ -23222,6 +23225,18 @@ INSERT INTO `item_mods` VALUES (18562,23,6);      -- Yhatdhara +1: Attack +6
 INSERT INTO `item_mods` VALUES (18562,48,9);      -- Weapon Skill Accuracy +9
 INSERT INTO `item_mods` VALUES (18564,8,10);      -- Devilish Scythe: STR+10
 INSERT INTO `item_mods` VALUES (18565,8,15);      -- Adflictio: STR+15
+INSERT INTO `item_mods` VALUES (18566,8,15); -- Crepuscular Scythe
+INSERT INTO `item_mods` VALUES (18566,12,15);
+INSERT INTO `item_mods` VALUES (18566,13,15);
+INSERT INTO `item_mods` VALUES (18566,23,55);
+INSERT INTO `item_mods` VALUES (18566,25,40);
+INSERT INTO `item_mods` VALUES (18566,30,40);
+INSERT INTO `item_mods` VALUES (18566,431,5);
+INSERT INTO `item_mods` VALUES (18566,499,12);
+INSERT INTO `item_mods` VALUES (18566,501,10);
+INSERT INTO `item_mods` VALUES (18566,951,0);
```

Comment:
0?

I know it's a shitton of work, but it would be really useful if you added a comment that states what the mod is next to the line

Reply:
I forgot to update it and I can do each line to update them